### ISSUE TITLE
Error installing gprmax

### ISSUE THREAD

**Issue by tidusquall17** [conda/conda#2258]
After "conda env create -f conda_env.yml" I have this error

```
C:\Users\Vincenzo\Desktop\gprMax-v.3.0.0b18>conda env create -f conda_env.yml
Fetching package metadata: ....
Solving package specifications: ..........
Fetching packages ...
mkl-11.3.1-0.t 100% |###############################| Time: 0:02:32 740.47 kB/s
jpeg-8d-vc14_0 100% |###############################| Time: 0:00:00 203.14 kB/s
libpng-1.6.17- 100% |###############################| Time: 0:00:01 317.79 kB/s
tk-8.5.18-vc14 100% |###############################| Time: 0:00:04 528.03 kB/s
zlib-1.2.8-vc1 100% |###############################| Time: 0:00:00 212.56 kB/s
hdf5-1.8.15.1- 100% |###############################| Time: 0:00:02 661.58 kB/s
libtiff-4.0.6- 100% |###############################| Time: 0:00:01 428.76 kB/s
cython-0.23.4- 100% |###############################| Time: 0:00:02 665.65 kB/s
numpy-1.10.4-p 100% |###############################| Time: 0:00:03 747.20 kB/s
psutil-4.1.0-p 100% |###############################| Time: 0:00:00 281.36 kB/s
pyparsing-2.0. 100% |###############################| Time: 0:00:00 182.66 kB/s
pytz-2015.7-py 100% |###############################| Time: 0:00:00 272.11 kB/s
qt-4.8.7-vc14_ 100% |###############################| Time: 0:01:10 771.13 kB/s
sip-4.16.9-py3 100% |###############################| Time: 0:00:00 283.95 kB/s
six-1.10.0-py3 100% |###############################| Time: 0:00:00 628.76 kB/s
cycler-0.10.0- 100% |###############################| Time: 0:00:00 781.07 kB/s
h5py-2.5.0-np1 100% |###############################| Time: 0:00:01 366.48 kB/s
pyqt-4.11.4-py 100% |###############################| Time: 0:00:05 697.00 kB/s
python-dateuti 100% |###############################| Time: 0:00:00 299.41 kB/s
matplotlib-1.5 100% |###############################| Time: 0:00:08 735.14 kB/s
Extracting packages ...
[      COMPLETE      ]|##################################################| 100%
Linking packages ...
[      COMPLETE      ]|##################################################| 100%
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "C:\Users\Vincenzo\Miniconda3\Scripts\conda-env-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\Users\Vincenzo\Miniconda3\lib\site-packages\conda_env\cli\main.py", line 68, in main
    return args_func(args, parser)
  File "C:\Users\Vincenzo\Miniconda3\lib\site-packages\conda\cli\main.py", line 146, in args_func
    args.func(args, p)
  File "C:\Users\Vincenzo\Miniconda3\lib\site-packages\conda_env\cli\main_create.py", line 100, in execute
    installer.install(prefix, pkg_specs, args, env)
  File "C:\Users\Vincenzo\Miniconda3\lib\site-packages\conda_env\installers\pip.py", line 8, in install
    pip_cmd = pip_args(prefix) + ['install', ] + specs
TypeError: unsupported operand type(s) for +: 'NoneType' and 'list'

```

Can you help me?

**Comment by hankhank** [comment]
Had the same issue 

Hack to work around is to edit following file

Anaconda2/Lib/site-packages/conda_env/installers/pip.py:8
pip_cmd = ["c:\Anaconda2\Scripts\pip.exe", ] + ['install', ] + specs

**Comment by jakirkham** [comment]
@tidusquall17, could you please share `conda_env.yml` in a gist and link it here?

**Comment by tidusquall17** [comment]
@hankhank i tried with this

```
C:\Users\Vincenzo>miniconda3\Scripts\pip.exe install specs
Collecting specs
  Could not find a version that satisfies the requirement specs (from versions: )
No matching distribution found for specs

```

but i don't undestand if it's what you meant


@jakirkham this is the link of my conda_env.yml
https://www.dropbox.com/s/zhc9epytjqooi1z/conda_env.yml?dl=0


Please try to explain simpler than you can thank you

**Comment by jakirkham** [comment]
Versions of `conda`, `conda env`, etc.?

**Comment by tidusquall17** [comment]
conda 4.0.4 
conda_env 2.4.5

**Comment by tidusquall17** [comment]
Nobody can help me?

**Comment by kalefranz** [comment]
fixed in `4.0.5`
